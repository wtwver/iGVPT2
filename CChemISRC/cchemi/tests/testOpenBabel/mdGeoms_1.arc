

                     SUMMARY OF PM6 CALCULATION, Site No: 1079

                                                       MOPAC2012 (Version: 15.146L)
                                                       Sat Dec 19 17:49:11 2015
                                                       No. of days left = 158

           Empirical Formula: C6 H14 N O5  =    26 atoms

PM6 CHARGE=1 Singlet

 Quantum Mechanics Energy(kCal/mol) =0.000000


     GEOMETRY OPTIMISED USING EIGENVECTOR FOLLOWING (EF).     
     SCF FIELD WAS ACHIEVED                                   

          HEAT OF FORMATION       =        -90.23764 KCAL/MOL =    -377.55427 KJ/MOL
          TOTAL ENERGY            =      -2557.07528 EV
          ELECTRONIC ENERGY       =     -14494.82446 EV
          CORE-CORE REPULSION     =      11937.74918 EV
          GRADIENT NORM           =          0.90507
          DIPOLE                  =          3.65223 DEBYE    POINT GROUP:       C1  
          NO. OF FILLED LEVELS    =         36
          CHARGE ON SYSTEM        =          1
          IONIZATION POTENTIAL    =         14.286741 EV
          HOMO LUMO ENERGIES (EV) =        -14.287 -3.670
          MOLECULAR WEIGHT        =        180.180
          COSMO AREA              =        186.79 SQUARE ANGSTROMS
          COSMO VOLUME            =        197.32 CUBIC ANGSTROMS

          MOLECULAR DIMENSIONS (Angstroms)

            Atom       Atom       Distance
            H    17    H    24     6.16455
            H    21    H    15     5.63504
            H    25    H    14     3.99620
          SCF CALCULATIONS        =         65
          WALL-CLOCK TIME         =          1.062 SECONDS
          COMPUTATION TIME        =         15.501 SECONDS


          FINAL GEOMETRY OBTAINED
PM6 CHARGE=1 Singlet

 Quantum Mechanics Energy(kCal/mol) =0.000000
  C     1.14972381 +1   0.59528340 +1   0.76558205 +1
  O    -0.04216233 +1   1.30564264 +1   0.32014803 +1
  C    -1.26757727 +1   0.72908547 +1   0.77975516 +1
  O    -2.25553693 +1   1.62944401 +1   0.31330786 +1
  C    -1.53689404 +1  -0.62679236 +1   0.06098517 +1
  N    -2.24508551 +1  -0.38777219 +1  -1.26318931 +1
  C    -0.19839079 +1  -1.39013592 +1  -0.22466165 +1
  O     0.10843442 +1  -1.19637326 +1  -1.59631098 +1
  C     0.94230761 +1  -0.93899398 +1   0.71775166 +1
  O     0.50863304 +1  -1.30873184 +1   2.02980788 +1
  C     2.24252766 +1   1.11173577 +1  -0.17524947 +1
  O     2.40216650 +1   0.10889610 +1  -1.20379262 +1
  H     1.33390337 +1   0.91496361 +1   1.82342818 +1
  H    -1.28870491 +1   0.63790043 +1   1.88808092 +1
  H    -1.90443528 +1   2.56489812 +1   0.30168791 +1
  H    -2.18511908 +1  -1.25624639 +1   0.73086168 +1
  H    -2.93125519 +1  -1.12968938 +1  -1.45683422 +1
  H    -0.36695917 +1  -2.49228219 +1  -0.12574030 +1
  H     0.88696867 +1  -0.53778836 +1  -1.72524000 +1
  H     1.89436228 +1  -1.45449927 +1   0.43587520 +1
  H     1.10419681 +1  -1.98680869 +1   2.42883943 +1
  H     3.21431381 +1   1.22357725 +1   0.34639637 +1
  H     1.94743321 +1   2.07668045 +1  -0.63807682 +1
  H     3.01527099 +1   0.43171550 +1  -1.90685459 +1
  H    -1.52498713 +1  -0.38691289 +1  -2.03982430 +1
  H    -2.73009305 +1   0.54089129 +1  -1.24957495 +1
 
