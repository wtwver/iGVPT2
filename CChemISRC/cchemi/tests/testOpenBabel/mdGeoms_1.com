#P B3LYP/aug-cc-pvdz
#  Units(Ang,Deg)

File generated by Gabedit
Quantum Mechanics Energy(kCal/mol) = 74.548134

1 1
C 1.128713 0.580785 0.731760
O -0.031821 1.284569 0.231607
C -1.245485 0.742485 0.724212
O -2.307642 1.609036 0.317590
C -1.505892 -0.639212 0.067789
N -2.190707 -0.397126 -1.261588
C -0.192618 -1.391983 -0.222333
O 0.141620 -1.192568 -1.610314
C 0.899830 -0.952455 0.767506
O 0.412777 -1.309205 2.071259
C 2.327935 0.967364 -0.148338
O 2.251443 0.312500 -1.416737
H 1.312421 0.962367 1.743877
H -1.237694 0.685655 1.818660
H -2.183931 2.424281 0.842601
H -2.213561 -1.212921 0.674722
H -2.860227 -1.122598 -1.522786
H -0.383116 -2.465320 -0.104403
H 1.048118 -0.802041 -1.664632
H 1.832877 -1.500724 0.597016
H 1.184678 -1.413945 2.663514
H 3.264482 0.662118 0.328550
H 2.344262 2.047735 -0.324634
H 2.972713 0.691543 -1.960449
H -1.440497 -0.368498 -1.975415
H -2.640862 0.532545 -1.222744

