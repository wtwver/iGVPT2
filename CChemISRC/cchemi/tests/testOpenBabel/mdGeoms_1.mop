PM6 CHARGE=1 Singlet

 Quantum Mechanics Energy(kCal/mol) =74.548134
 C 1.128713 1 0.580785 1 0.731760 1
 O -0.031821 1 1.284569 1 0.231607 1
 C -1.245485 1 0.742485 1 0.724212 1
 O -2.307642 1 1.609036 1 0.317590 1
 C -1.505892 1 -0.639212 1 0.067789 1
 N -2.190707 1 -0.397126 1 -1.261588 1
 C -0.192618 1 -1.391983 1 -0.222333 1
 O 0.141620 1 -1.192568 1 -1.610314 1
 C 0.899830 1 -0.952455 1 0.767506 1
 O 0.412777 1 -1.309205 1 2.071259 1
 C 2.327935 1 0.967364 1 -0.148338 1
 O 2.251443 1 0.312500 1 -1.416737 1
 H 1.312421 1 0.962367 1 1.743877 1
 H -1.237694 1 0.685655 1 1.818660 1
 H -2.183931 1 2.424281 1 0.842601 1
 H -2.213561 1 -1.212921 1 0.674722 1
 H -2.860227 1 -1.122598 1 -1.522786 1
 H -0.383116 1 -2.465320 1 -0.104403 1
 H 1.048118 1 -0.802041 1 -1.664632 1
 H 1.832877 1 -1.500724 1 0.597016 1
 H 1.184678 1 -1.413945 1 2.663514 1
 H 3.264482 1 0.662118 1 0.328550 1
 H 2.344262 1 2.047735 1 -0.324634 1
 H 2.972713 1 0.691543 1 -1.960449 1
 H -1.440497 1 -0.368498 1 -1.975415 1
 H -2.640862 1 0.532545 1 -1.222744 1
