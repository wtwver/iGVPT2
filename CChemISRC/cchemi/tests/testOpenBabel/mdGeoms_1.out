 *******************************************************************************
 ** Site#: 1079         For non-commercial use only     Version 15.146L 64BITS**
 *******************************************************************************
 ** Cite this work as:                                                        **
 **                                                                           **
 ** (a) MOPAC2012, James J. P. Stewart, Stewart Computational Chemistry,      **
 **  Version 15.146L web: HTTP://OpenMOPAC.net         Days remaining: 158    **
 **                                                                           **
 **                                 AND                                       **
 **                                                                           **
 ** (b) Maia, J.D.C., et al., J. Chem. Theory Comput. 2012, 8, 3072-3081.     **
 **                DOI: 10.1021/ct3004645.                                    **
 *******************************************************************************
 **                                                                           **
 **                                MOPAC2012                                  **
 **                                                                           **
 *******************************************************************************

                            PM6 CALCULATION RESULTS


 *******************************************************************************
 *  CALCULATION DONE:                                Sat Dec 19 17:49:10 2015  *
 *  PM6        - The PM6 Hamiltonian to be used
 *  SINGLET    - SPIN STATE DEFINED AS A SINGLET
 *
 *
 *
 *                 CHARGE ON SYSTEM =     1
 *
 *
 *
 *  T=         - A TIME OF 172800.0 SECONDS REQUESTED
 *  DUMP=N     - RESTART FILE WRITTEN EVERY  7200.000 SECONDS
 *******************************************************************************
PM6 CHARGE=1 Singlet

 Quantum Mechanics Energy(kCal/mol) =0.000000
   ATOM   CHEMICAL          X               Y               Z
  NUMBER   SYMBOL      (ANGSTROMS)     (ANGSTROMS)     (ANGSTROMS)
 
     1       C          1.12875400  *   0.58074200  *   0.73182100  *
     2       O         -0.03176800  *   1.28455400  *   0.23168000  *
     3       C         -1.24545700  *   0.74247700  *   0.72420200  *
     4       O         -2.30756100  *   1.60905500  *   0.31749300  *
     5       C         -1.50586000  *  -0.63922100  *   0.06777700  *
     6       N         -2.19065000  *  -0.39713800  *  -1.26161500  *
     7       C         -0.19258900  *  -1.39201700  *  -0.22229600  *
     8       O          0.14167200  *  -1.19266300  *  -1.61027900  *
     9       C          0.89983700  *  -0.95249100  *   0.76757100  *
    10       O          0.41271000  *  -1.30920800  *   2.07131200  *
    11       C          2.32793400  *   0.96732900  *  -0.14832800  *
    12       O          2.25124100  *   0.31266000  *  -1.41681500  *
    13       H          1.31249600  *   0.96231300  *   1.74393700  *
    14       H         -1.23774900  *   0.68565600  *   1.81865400  *
    15       H         -2.18379900  *   2.42433700  *   0.84243500  *
    16       H         -2.21354300  *  -1.21291900  *   0.67470300  *
    17       H         -2.86026500  *  -1.12253800  *  -1.52277400  *
    18       H         -0.38311100  *  -2.46534800  *  -0.10434300  *
    19       H          1.04813800  *  -0.80205900  *  -1.66459400  *
    20       H          1.83288700  *  -1.50077800  *   0.59713200  *
    21       H          1.18456700  *  -1.41396500  *   2.66361900  *
    22       H          3.26451100  *   0.66193900  *   0.32841400  *
    23       H          2.34433000  *   2.04772200  *  -0.32447400  *
    24       H          2.97224300  *   0.69199100  *  -1.96068100  *
    25       H         -1.44044200  *  -0.36862400  *  -1.97545000  *
    26       H         -2.64071000  *   0.53258200  *  -1.22281100  *
 


          CARTESIAN COORDINATES 

    NO.       ATOM         X         Y         Z

     1         C        1.1288    0.5807    0.7318
     2         O       -0.0318    1.2846    0.2317
     3         C       -1.2455    0.7425    0.7242
     4         O       -2.3076    1.6091    0.3175
     5         C       -1.5059   -0.6392    0.0678
     6         N       -2.1906   -0.3971   -1.2616
     7         C       -0.1926   -1.3920   -0.2223
     8         O        0.1417   -1.1927   -1.6103
     9         C        0.8998   -0.9525    0.7676
    10         O        0.4127   -1.3092    2.0713
    11         C        2.3279    0.9673   -0.1483
    12         O        2.2512    0.3127   -1.4168
    13         H        1.3125    0.9623    1.7439
    14         H       -1.2377    0.6857    1.8187
    15         H       -2.1838    2.4243    0.8424
    16         H       -2.2135   -1.2129    0.6747
    17         H       -2.8603   -1.1225   -1.5228
    18         H       -0.3831   -2.4653   -0.1043
    19         H        1.0481   -0.8021   -1.6646
    20         H        1.8329   -1.5008    0.5971
    21         H        1.1846   -1.4140    2.6636
    22         H        3.2645    0.6619    0.3284
    23         H        2.3443    2.0477   -0.3245
    24         H        2.9722    0.6920   -1.9607
    25         H       -1.4404   -0.3686   -1.9754
    26         H       -2.6407    0.5326   -1.2228

     ===================== shared-memory parallel CPU Section ======================
     **                                                                           **
     **  MOPAC calculations will run in shared-memory parallel CPUs               **
     **                                                                           **
     **  Number of Threads = 16                                                   **
     **                                                                           **
     **                                                                           **
     **  For information on this work, see:                                       **
     **                                                                           **
     **  Maia, J.D.C., et al., J. Chem. Theory Comput. 2012, 8, 3072-3081.        **
     **        DOI: 10.1021/ct3004645.                                            **
     **                                                                           **
     ===================== shared-memory parallel CPU Section ======================
      


 General Reference for PM6:
 "Optimization of Parameters for Semiempirical Methods V: Modification of NDDO Approximations
 and Application to 70 Elements", J. J. P. Stewart, J. Mol. Mod., 13, 1173-1213 (2007)
 URL: http://www.springerlink.com/content/ar33482301010477/fulltext.pdf

           Empirical Formula: C6 H14 N O5  =    26 atoms



      MOLECULAR POINT GROUP   :   C1  


 SINGLET STATE CALCULATION


      RHF CALCULATION, NO. OF DOUBLY OCCUPIED LEVELS =   36

      Geometry optimization using EF

          DIAGONAL MATRIX USED AS START HESSIAN

 CYCLE:     1 TIME:   0.062 TIME LEFT:  2.00D  GRAD.:    82.919 HEAT: -77.75474
 CYCLE:     2 TIME:   0.016 TIME LEFT:  2.00D  GRAD.:   124.106 HEAT: -78.84164
 CYCLE:     3 TIME:   0.023 TIME LEFT:  2.00D  GRAD.:    62.631 HEAT: -83.27647
 CYCLE:     4 TIME:   0.023 TIME LEFT:  2.00D  GRAD.:    35.274 HEAT: -84.66331
 CYCLE:     5 TIME:   0.016 TIME LEFT:  2.00D  GRAD.:    43.254 HEAT: -85.11129
 CYCLE:     6 TIME:   0.012 TIME LEFT:  2.00D  GRAD.:    45.040 HEAT: -85.39839
 CYCLE:     7 TIME:   0.016 TIME LEFT:  2.00D  GRAD.:    35.766 HEAT: -85.80192
 CYCLE:     8 TIME:   0.023 TIME LEFT:  2.00D  GRAD.:    22.515 HEAT: -86.21661
 CYCLE:     9 TIME:   0.016 TIME LEFT:  2.00D  GRAD.:    27.852 HEAT: -86.32712
 CYCLE:    10 TIME:   0.023 TIME LEFT:  2.00D  GRAD.:    30.376 HEAT: -86.54367
 CYCLE:    11 TIME:   0.016 TIME LEFT:  2.00D  GRAD.:    31.882 HEAT: -86.68277
 CYCLE:    12 TIME:   0.012 TIME LEFT:  2.00D  GRAD.:    19.934 HEAT: -87.06414
 CYCLE:    13 TIME:   0.016 TIME LEFT:  2.00D  GRAD.:    27.706 HEAT: -87.16552
 CYCLE:    14 TIME:   0.016 TIME LEFT:  2.00D  GRAD.:    32.786 HEAT: -87.22532
 CYCLE:    15 TIME:   0.012 TIME LEFT:  2.00D  GRAD.:    27.448 HEAT: -87.47306
 CYCLE:    16 TIME:   0.016 TIME LEFT:  2.00D  GRAD.:    19.730 HEAT: -87.73183
 CYCLE:    17 TIME:   0.023 TIME LEFT:  2.00D  GRAD.:    17.377 HEAT: -87.83835
 CYCLE:    18 TIME:   0.016 TIME LEFT:  2.00D  GRAD.:    13.887 HEAT: -87.93772
 CYCLE:    19 TIME:   0.012 TIME LEFT:  2.00D  GRAD.:     8.943 HEAT: -88.07151
 CYCLE:    20 TIME:   0.031 TIME LEFT:  2.00D  GRAD.:     9.484 HEAT: -88.22469
 CYCLE:    21 TIME:   0.016 TIME LEFT:  2.00D  GRAD.:    11.427 HEAT: -88.40771
 CYCLE:    22 TIME:   0.016 TIME LEFT:  2.00D  GRAD.:    14.210 HEAT: -88.60618
 CYCLE:    23 TIME:   0.012 TIME LEFT:  2.00D  GRAD.:    11.230 HEAT: -88.76840
 CYCLE:    24 TIME:   0.016 TIME LEFT:  2.00D  GRAD.:    13.307 HEAT: -88.97645
 CYCLE:    25 TIME:   0.012 TIME LEFT:  2.00D  GRAD.:    15.602 HEAT: -89.26176
 CYCLE:    26 TIME:   0.016 TIME LEFT:  2.00D  GRAD.:    12.510 HEAT: -89.40612
 CYCLE:    27 TIME:   0.016 TIME LEFT:  2.00D  GRAD.:     7.467 HEAT: -89.51112
 CYCLE:    28 TIME:   0.012 TIME LEFT:  2.00D  GRAD.:     7.260 HEAT: -89.57278
 CYCLE:    29 TIME:   0.027 TIME LEFT:  2.00D  GRAD.:     5.957 HEAT: -89.63623
 CYCLE:    30 TIME:   0.016 TIME LEFT:  2.00D  GRAD.:     5.140 HEAT: -89.70180
 CYCLE:    31 TIME:   0.016 TIME LEFT:  2.00D  GRAD.:     5.176 HEAT: -89.75168
 CYCLE:    32 TIME:   0.012 TIME LEFT:  2.00D  GRAD.:     5.673 HEAT: -89.80360
 CYCLE:    33 TIME:   0.016 TIME LEFT:  2.00D  GRAD.:     4.502 HEAT: -89.84861
 CYCLE:    34 TIME:   0.012 TIME LEFT:  2.00D  GRAD.:     4.819 HEAT: -89.88356
 CYCLE:    35 TIME:   0.012 TIME LEFT:  2.00D  GRAD.:     6.119 HEAT: -89.90406
 CYCLE:    36 TIME:   0.016 TIME LEFT:  2.00D  GRAD.:     3.920 HEAT: -89.93412
 CYCLE:    37 TIME:   0.016 TIME LEFT:  2.00D  GRAD.:     2.962 HEAT: -89.95570
 CYCLE:    38 TIME:   0.012 TIME LEFT:  2.00D  GRAD.:     3.180 HEAT: -89.98287
 CYCLE:    39 TIME:   0.016 TIME LEFT:  2.00D  GRAD.:     2.977 HEAT: -90.00137
 CYCLE:    40 TIME:   0.023 TIME LEFT:  2.00D  GRAD.:     3.177 HEAT: -90.01600
 CYCLE:    41 TIME:   0.016 TIME LEFT:  2.00D  GRAD.:     2.872 HEAT: -90.02689
 CYCLE:    42 TIME:   0.016 TIME LEFT:  2.00D  GRAD.:     2.323 HEAT: -90.03602
 CYCLE:    43 TIME:   0.012 TIME LEFT:  2.00D  GRAD.:     2.251 HEAT: -90.04585
 CYCLE:    44 TIME:   0.016 TIME LEFT:  2.00D  GRAD.:     2.564 HEAT: -90.05751
 CYCLE:    45 TIME:   0.012 TIME LEFT:  2.00D  GRAD.:     2.939 HEAT: -90.07596
 CYCLE:    46 TIME:   0.016 TIME LEFT:  2.00D  GRAD.:     3.210 HEAT: -90.09451
 CYCLE:    47 TIME:   0.012 TIME LEFT:  2.00D  GRAD.:     3.046 HEAT: -90.11326
 CYCLE:    48 TIME:   0.016 TIME LEFT:  2.00D  GRAD.:     3.201 HEAT: -90.13060
 CYCLE:    49 TIME:   0.016 TIME LEFT:  2.00D  GRAD.:     3.043 HEAT: -90.14878
 CYCLE:    50 TIME:   0.012 TIME LEFT:  2.00D  GRAD.:     2.368 HEAT: -90.16532
 CYCLE:    51 TIME:   0.027 TIME LEFT:  2.00D  GRAD.:     2.605 HEAT: -90.18067
 CYCLE:    52 TIME:   0.016 TIME LEFT:  2.00D  GRAD.:     1.775 HEAT: -90.19034
 CYCLE:    53 TIME:   0.016 TIME LEFT:  2.00D  GRAD.:     1.708 HEAT: -90.19897
 CYCLE:    54 TIME:   0.012 TIME LEFT:  2.00D  GRAD.:     1.627 HEAT: -90.20595
 CYCLE:    55 TIME:   0.016 TIME LEFT:  2.00D  GRAD.:     1.614 HEAT: -90.21255
 CYCLE:    56 TIME:   0.012 TIME LEFT:  2.00D  GRAD.:     1.621 HEAT: -90.21875
 CYCLE:    57 TIME:   0.016 TIME LEFT:  2.00D  GRAD.:     1.725 HEAT: -90.22415
 CYCLE:    58 TIME:   0.016 TIME LEFT:  2.00D  GRAD.:     1.457 HEAT: -90.22964
 CYCLE:    59 TIME:   0.012 TIME LEFT:  2.00D  GRAD.:     1.151 HEAT: -90.23445
 CYCLE:    60 TIME:   0.016 TIME LEFT:  2.00D  GRAD.:     0.905 HEAT: -90.23764

     GRADIENT =  0.90507  IS LESS THAN CUTOFF =  1.00000



 -------------------------------------------------------------------------------
PM6 CHARGE=1 Singlet

 Quantum Mechanics Energy(kCal/mol) =0.000000


     GEOMETRY OPTIMISED USING EIGENVECTOR FOLLOWING (EF).     
     SCF FIELD WAS ACHIEVED                                   


                              PM6 CALCULATION
                                                       MOPAC2012 (Version: 15.146L)
                                                       Sat Dec 19 17:49:11 2015
                                                       No. of days left = 158




          FINAL HEAT OF FORMATION =        -90.23764 KCAL/MOL =    -377.55427 KJ/MOL


          TOTAL ENERGY            =      -2557.07528 EV
          ELECTRONIC ENERGY       =     -14494.82446 EV  POINT GROUP:     C1  
          CORE-CORE REPULSION     =      11937.74918 EV
          COSMO AREA              =        186.79 SQUARE ANGSTROMS
          COSMO VOLUME            =        197.32 CUBIC ANGSTROMS

          GRADIENT NORM           =          0.90507
          IONIZATION POTENTIAL    =         14.286741 EV
          HOMO LUMO ENERGIES (EV) =        -14.287 -3.670
          NO. OF FILLED LEVELS    =         36
          MOLECULAR WEIGHT        =        180.180

          MOLECULAR DIMENSIONS (Angstroms)

            Atom       Atom       Distance
            H    17    H    24     6.16455
            H    21    H    15     5.63504
            H    25    H    14     3.99620


          SCF CALCULATIONS        =         65
 
          WALL-CLOCK TIME         =          1.059 SECONDS
          COMPUTATION TIME        =         15.457 SECONDS




   ATOM   CHEMICAL          X               Y               Z
  NUMBER   SYMBOL      (ANGSTROMS)     (ANGSTROMS)     (ANGSTROMS)
 
     1       C          1.14972381  *   0.59528340  *   0.76558205  *
     2       O         -0.04216233  *   1.30564264  *   0.32014803  *
     3       C         -1.26757727  *   0.72908547  *   0.77975516  *
     4       O         -2.25553693  *   1.62944401  *   0.31330786  *
     5       C         -1.53689404  *  -0.62679236  *   0.06098517  *
     6       N         -2.24508551  *  -0.38777219  *  -1.26318931  *
     7       C         -0.19839079  *  -1.39013592  *  -0.22466165  *
     8       O          0.10843442  *  -1.19637326  *  -1.59631098  *
     9       C          0.94230761  *  -0.93899398  *   0.71775166  *
    10       O          0.50863304  *  -1.30873184  *   2.02980788  *
    11       C          2.24252766  *   1.11173577  *  -0.17524947  *
    12       O          2.40216650  *   0.10889610  *  -1.20379262  *
    13       H          1.33390337  *   0.91496361  *   1.82342818  *
    14       H         -1.28870491  *   0.63790043  *   1.88808092  *
    15       H         -1.90443528  *   2.56489812  *   0.30168791  *
    16       H         -2.18511908  *  -1.25624639  *   0.73086168  *
    17       H         -2.93125519  *  -1.12968938  *  -1.45683422  *
    18       H         -0.36695917  *  -2.49228219  *  -0.12574030  *
    19       H          0.88696867  *  -0.53778836  *  -1.72524000  *
    20       H          1.89436228  *  -1.45449927  *   0.43587520  *
    21       H          1.10419681  *  -1.98680869  *   2.42883943  *
    22       H          3.21431381  *   1.22357725  *   0.34639637  *
    23       H          1.94743321  *   2.07668045  *  -0.63807682  *
    24       H          3.01527099  *   0.43171550  *  -1.90685459  *
    25       H         -1.52498713  *  -0.38691289  *  -2.03982430  *
    26       H         -2.73009305  *   0.54089129  *  -1.24957495  *

                             CARTESIAN COORDINATES

   1    C        1.149723814     0.595283401     0.765582053
   2    O       -0.042162335     1.305642642     0.320148030
   3    C       -1.267577274     0.729085474     0.779755157
   4    O       -2.255536926     1.629444005     0.313307856
   5    C       -1.536894035    -0.626792359     0.060985173
   6    N       -2.245085508    -0.387772186    -1.263189308
   7    C       -0.198390794    -1.390135920    -0.224661647
   8    O        0.108434419    -1.196373264    -1.596310977
   9    C        0.942307612    -0.938993978     0.717751657
  10    O        0.508633045    -1.308731838     2.029807880
  11    C        2.242527664     1.111735774    -0.175249466
  12    O        2.402166500     0.108896103    -1.203792624
  13    H        1.333903370     0.914963611     1.823428184
  14    H       -1.288704912     0.637900429     1.888080921
  15    H       -1.904435275     2.564898121     0.301687909
  16    H       -2.185119085    -1.256246387     0.730861677
  17    H       -2.931255187    -1.129689377    -1.456834224
  18    H       -0.366959168    -2.492282185    -0.125740303
  19    H        0.886968673    -0.537788364    -1.725239996
  20    H        1.894362282    -1.454499269     0.435875195
  21    H        1.104196814    -1.986808690     2.428839433
  22    H        3.214313810     1.223577252     0.346396368
  23    H        1.947433214     2.076680452    -0.638076825
  24    H        3.015270995     0.431715501    -1.906854593
  25    H       -1.524987127    -0.386912893    -2.039824300
  26    H       -2.730093051     0.540891292    -1.249574945


           Empirical Formula: C6 H14 N O5  =    26 atoms



      MOLECULAR POINT GROUP   :   C1  


                  EIGENVALUES  
 -38.54525 -35.07756 -34.54650 -33.74909 -33.40127 -33.27046 -31.01602 -30.74460
 -28.44066 -27.59804 -25.80907 -24.28131 -23.82684 -23.65990 -22.94637 -22.53390
 -22.22973 -21.59491 -20.97790 -20.57533 -20.41274 -19.18589 -18.93362 -18.76680
 -18.45387 -18.01016 -17.79757 -17.08309 -17.03877 -16.57554 -16.19617 -15.59287
 -15.16508 -14.69554 -14.66121 -14.28674  -3.66998  -2.65613  -2.16179  -2.08869
  -1.90279  -1.67235  -1.50889  -1.19358  -1.16082  -0.97164  -0.79215  -0.62299
  -0.05555   0.04380   0.33258   0.59144   0.93786   1.02447   1.13492   1.27651
   1.32526   1.45649   1.59394   1.72237   1.85177   2.06307


              NET ATOMIC CHARGES AND DIPOLE CONTRIBUTIONS

  ATOM NO.   TYPE          CHARGE      No. of ELECS.   s-Pop       p-Pop       
    1          C           0.106224        3.8938     1.12718     2.76660
    2          O          -0.512582        6.5126     1.85994     4.65264
    3          C           0.386166        3.6138     1.11621     2.49763
    4          O          -0.567288        6.5673     1.81664     4.75064
    5          C          -0.242754        4.2428     1.15361     3.08914
    6          N          -0.076867        5.0769     1.40465     3.67222
    7          C           0.142821        3.8572     1.11606     2.74112
    8          O          -0.576697        6.5767     1.82949     4.74720
    9          C           0.032465        3.9675     1.12435     2.84318
   10          O          -0.548023        6.5480     1.82064     4.72738
   11          C          -0.050040        4.0500     1.11355     2.93649
   12          O          -0.568123        6.5681     1.81506     4.75306
   13          H           0.171111        0.8289     0.82889
   14          H           0.164690        0.8353     0.83531
   15          H           0.369979        0.6300     0.63002
   16          H           0.205832        0.7942     0.79417
   17          H           0.253780        0.7462     0.74622
   18          H           0.170819        0.8292     0.82918
   19          H           0.372321        0.6277     0.62768
   20          H           0.180331        0.8197     0.81967
   21          H           0.341909        0.6581     0.65809
   22          H           0.160166        0.8398     0.83983
   23          H           0.164997        0.8350     0.83500
   24          H           0.343510        0.6565     0.65649
   25          H           0.290910        0.7091     0.70909
   26          H           0.284341        0.7157     0.71566
 DIPOLE           X         Y         Z       TOTAL
 POINT-CHG.    -2.694    -0.762    -1.390     3.126
 HYBRID         0.007     0.522    -1.072     1.193
 SUM           -2.687    -0.240    -2.462     3.652


          ATOMIC ORBITAL ELECTRON POPULATIONS

     Atom    s        px        py        pz   
    1  C   1.12718   0.79843   0.91557   1.05260
    2  O   1.85994   1.25520   1.59727   1.80016
    3  C   1.11621   0.70823   0.79335   0.99604
    4  O   1.81664   1.50011   1.35726   1.89328
    5  C   1.15361   1.07602   1.11199   0.90113
    6  N   1.40465   1.23187   1.24106   1.19929
    7  C   1.11606   0.90192   1.05416   0.78504
    8  O   1.82949   1.66050   1.75103   1.33567
    9  C   1.12435   1.04446   0.99301   0.80571
   10  O   1.82064   1.64966   1.69766   1.38006
   11  C   1.11355   1.06475   0.95839   0.91335
   12  O   1.81506   1.75111   1.53564   1.46632
   13  H   0.82889
   14  H   0.83531
   15  H   0.63002
   16  H   0.79417
   17  H   0.74622
   18  H   0.82918
   19  H   0.62768
   20  H   0.81967
   21  H   0.65809
   22  H   0.83983
   23  H   0.83500
   24  H   0.65649
   25  H   0.70909
   26  H   0.71566

          JOB ENDED NORMALLY



 TOTAL JOB TIME:             1.06 SECONDS

 == MOPAC DONE ==
